ocs_ci/templates/logging-deployment/clusterlogging_operator/instance.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"  
+  logStore:
+    type: "elasticsearch"  
+    elasticsearch:
+      nodeCount: 3
+      resources:
+        limits:
+          memory: "2Gi"
+        requests:
+          cpu: "300m"
+          memory: "16Gi"
+      storage:
+        storageClassName: "gp2"
```

Comment:
I think gp2 is going to change as not the default one.
Please share this entire set of yamls with eng to get clear confirmation we are testing the right thing

Reply:
Waiting for the confirmation from the dev team. I will update the yaml once I get to know the default storageclass